modules/polling/context/BallotContext.tsx

```diff
@@ -109,26 +129,25 @@ export const BallotProvider = ({ children }: PropTypes): React.ReactElement => {
 
   // add vote to ballot
 
-  const addVoteToBallot = (pollId: number, ballotVote: BallotVote) => {
+  const addVoteToBallot = (pollId: number, ballotVote: Partial<BallotVote>) => {
     setTxId(null);
     setCommentSignature('');
     const newBallot = {
       ...ballot,
-      [pollId]: ballotVote
+      [pollId]: {
+        ...ballotVote,
+        timestamp: Date.now()
```

Comment:
I don't think you're using timestamp for anything, can remove it here and in `updateVoteFromBallot` and from the type def